### ISSUE TITLE
[FEATURE]

### ISSUE THREAD

**Issue by zasnaz** [booksaw/BetterTeams#7]
Hello
Can you make the team mates appear with same color like green for example,
![explain](https://user-images.githubusercontent.com/66457588/85425704-2d621500-b582-11ea-8305-427d565538a2.png)
Like everyone inside [sssss] sees the color of the team is green or uniform color, other teams appears in red color or something so team members can distinguish,
Just a simple suggestion sorry for disturb.

**Comment by booksaw** [comment]
The feature itself is very simple, and I like the idea. 
But before I implement the feature into the next update, I need to make you aware of a few things: 
1. To set per player colours, I would probably override your servers chat manager, so any features that you use from there may not be available. 
2. The feature may not only override your chat management plugin. It may also require the plugin to be removed (this will vary based on the server setup), which means all other features of that plugin would be disabled

If you still want this feature after those warnings I can add it, but I figured I would tell you what problems may occur due to it.

**Comment by zasnaz** [comment]
I have an idea,
I personally use Placeholders to make the team name's appear, Placeholder does not affect on other plugins i think is that right ?
so i use %betterTeams_name% as prefix in my server, My suggestion is adding a new placeholder for allies maybe like %betterTeams_alliesname% this should work same as %betterTeams_name% but the features is the specialized with allies or members of the team so we can change the color of  %betterTeams_alliesname% but keep the %betterTeams_name%
I don't know if you understand my idea or not but I hope you are not confused.

**Comment by booksaw** [comment]
I like that idea, the only problem is that placeholder API does not work like that. 
Technical reasoning: When placeholder API asks my plugin what the value of the placeholder is. It only provides 1 piece of information, the player (in this case, this will be the player who is sending the message). Sadly placeholder API will not provide the player that the message is going to. 
From how I see it, I think overriding the chat manager would be the only way of doing so (and sending an individual message to each user, instead of the chat in general) 

The impression I am getting is that you would not have a use for the feature if it has to override the chat manager. Am I correct?

**Comment by zasnaz** [comment]
Yes brother you right,honestly I use placeholders in nametags I don't use it in chat(I was, but I removed it for now), so Idk if you can add this feature for nametags only at least ?or is it the same idea? I hope you find a way to make this, Thank you.

**Comment by zasnaz** [comment]
and I have a question outside the topic idk if you can help me, I have codes for one of the plugins that i use, i find the codes on github, the codes are fix for a plugin problem i don't know how to apply these codes on the plugin, sorry because the question not on the right place but I really don't know how to fix this.

**Comment by booksaw** [comment]
With nametags, I may be able to make it work. I will try and add this feature into the next version, but it may not be possible. 
Secondly, message me on spigot: https://www.spigotmc.org/members/booksaw123.120955/ about the other problem.

**Comment by booksaw** [comment]
Reading through this again, I realise there were a few questions I forgot to ask:
1. When you say nametag, do you mean the text above the players name? 
2. If yes, are you using placeholder API to put it there or the better teams option to automate the process? 

If you are not using the better teams option to automate the process, this would be impossible. If not I think I could do this (potentially with reduced compatibility with other plugins which controls scoreboards)
